extensions/dig/src/index.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  Detail,
+  Icon,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  useNavigation,
+} from "@raycast/api";
+import { useState, useEffect } from "react";
+const { execSync } = require("child_process");
+
+//const execSync2 = require('child_process').execSync;
+//const execSync2 = execSync;
+//var execSync2 = require('execSync');
+
+export default function DigSearchResultsList() {
+  const [query, setQuery] = useState<null | string>(null);
+  const [state, setState] = useState<Result[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const { push } = useNavigation();
+
+  useEffect(() => {
+    async function fetch() {
+      if (!query) {
+        setState([]);
+        return;
+      }
+      setIsLoading(true);
+      const results = await digByQuery(query);
+      setState(results);
+      setIsLoading(false);
+    }
+    fetch();
+  }, [query]);
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder="Type a valid Hostname (ex. raycast.com or raycast.com mx)"
+      onSearchTextChange={(text) => setQuery(text)}
+      throttle
+    >
+      {state.map((result, idx) => (
+        <List.Item
+          id={idx.toString()}
+          key={idx}
+          title={result.title}
+          icon="icon.png"
+          accessories={[{ text: result.summary }]}
+          actions={
+            <ActionPanel>
+              <CopyToClipboardAction title="Copy Destination" content={result.summary} />
+              <OpenInBrowserAction url={result.url} />
+              <ActionPanel.Item
```

Comment:
Now using the right syntax
```suggestion
              <Action.CopyToClipboard title="Copy Destination" content={result.summary} />
              <Action.OpenInBrowser url={result.url} />
              <Action
```